sketch/gameManager.ts

```diff
@@ -101,6 +101,7 @@ class GameManager {
         if (this.isThisPressed(this.startButton) && !this.isGameRunning) {
             this.isGameRunning = true
             this.gamePage.resetParam()
+            soundEffects.swooshSound()
```

Comment:
do we need it here i think it will work otherwise also since i any create game status again here so neednt call it again ....check please :)

Reply:
one is for when the level changes and one is for when we press the start button:)